secure-biometric-interface-integrator/lib/standardConstant.js

```diff
@@ -0,0 +1,77 @@
+const states = {
+  LOADING: "LOADING",
+  LOADED: "LOADED",
+  ERROR: "ERROR",
+  AUTHENTICATING: "AUTHENTICATING",
+};
+
+const DeviceStateStatus = {
+  Ready: "READY",
+  "Not Ready": "NOTREADY",
+  Busy: "BUSY",
+  "Not Registered": "NOTREGISTERED",
+};
+
+const DeviceState = {
+  READY: {
+    value: "READY",
+    name: "Ready",
+    class: "ready",
+    symbol: "\u25CF",
+  },
+  NOTREADY: {
+    value: "NOTREADY",
+    name: "Not Ready",
+    class: "not-ready",
+    symbol: "\u25CF",
+  },
+  BUSY: {
+    value: "BUSY",
+    name: "Busy",
+    class: "busy",
+    symbol: "\u25CF",
+  },
+  NOTREGISTERED: {
+    value: "NOTREGISTERED",
+    name: "Not Registered",
+    class: "not-registered",
+    symbol: "\u25CE",
+  },
+};
+
+const DEFAULT_PROPS = {
+  buttonLabel: "scan_and_verify",
+  disable: false,
+  langCode: "en",
```

Comment:
are we not using the 3 letter language?

Reply:
No Sasi. We are using 2 letter language, and also in esignet's UI as well using 2 letter language

Reply:
hi @gsasikumar, added 3 letter language in the component, check this [044194a](https://github.com/mosip/esignet-plugins/pull/5/commits/044194a5882341fd950e519ef9b08f59144e6caa) commit